main/templates/godfather_form.html

```diff
@@ -0,0 +1,15 @@
+{% block content %}
```

Comment:
I think It's better to order .htmls by folders. In this case, /templates/godfather/godfather_form.html

Reply:
Ok 👍